parlai/core/params.py

```diff
@@ -210,6 +210,22 @@ def _format_action_invocation(self, action):
         args_string = self._format_args(action, default)
         return ', '.join(action.option_strings) + ' ' + args_string
 
+    def _get_help_string(self, action):
+        help = action.help
+        if '%(default)' not in action.help:
```

Comment:
what's this `%` doing here?

Reply:
This is the magic constant that is replaced with the default value in argparse. I copy-pasta'd this code directly from Cython source:

https://github.com/python/cpython/blob/5bbac8cbdf140ebce446ea4e7db2b20a5d7b8402/Lib/argparse.py#L691-L698